docs/CA-Protocol-Basics.md

```diff
@@ -34,16 +34,16 @@ to be terminated.
 ## Request - Response
 
 Each message is either a request or a response to a request that was previously sent within the same connection. Each request 
-comes with a unique message identifier and the response repeats the identifier so that the requestor can match the incoming response 
-to its corresponding request. The requestor is the one responsible for the uniqueness of request identifiers. The requestee 
+comes with a unique message identifier and the response repeats the identifier so that the requester can match the incoming response 
```

Comment:
i made it as a suggestion, because -er is more commonly used BUT it doesn't mean what you wrote is wrong. go ahead and use -or